.aspect/workflows/terraform/workflows.tf

```diff
@@ -71,10 +79,23 @@ module "aspect_workflows" {
       min_runners               = 0
       queue                     = "aspect-default"
       resource_type             = "default"
-      scale_out_factor          = 2
       scaling_polling_frequency = 1 # check for queued jobs every 60s
       warming                   = true
     }
+    small = {
```

Comment:
It's too bad it's such a lot of extra configuration to do the thing we wish was the default behavior for everyone